### ISSUE TITLE
Should we get Glass Jugs for Storing Juices/Smoothies in the Fridge?

### ISSUE THREAD

**Issue by nelsonic** [dwyl/health#17]
@iteles have you looked into getting Glass Jugs?
I'm quite happy with the kilner jars for portability, but _would_ it make sense to have glass jugs in the fridge to store juice or _sugar-free_-lemonade...?

http://www.amazon.co.uk/s/ref=nb_sb_noss_1?url=search-alias%3Daps&field-keywords=glass+fridge+jug

**Comment by iteles** [comment]
Agreed. Plus it would free up some kilner jars :wink: I'm sure we had more than we appear to have now.

I'll measure the space :+1:

**Comment by nelsonic** [comment]
Most of the jugs will have been measured by the manufacturers and will conform to a standard size (which is pretty much dictated by the CocaCola Company...) but yeah, best to measure to be sure.
thanks! :+1: 
#Day6 :open_mouth:

**Comment by iteles** [comment]
Our door shelves are 10cm wide, but the bottom one is 33cm tall and the middle shelf is 26cm tall.

That means that the [1 litre jug](http://www.amazon.co.uk/Luminarc-Glass-Quadro-Fridge-Litre/dp/B0009P5LQ4/ref=sr_1_1?ie=UTF8&qid=1442442129&sr=8-1&keywords=glass+fridge+jug) (**£2.49** each as an add-on), [1.7 litre](http://www.amazon.co.uk/Luminarc-Glass-Quadro-Fridge-Litre/dp/B002AMVQUE/ref=sr_1_2?ie=UTF8&qid=1442442367&sr=8-2&keywords=glass+fridge+jug) (**£6.17** each) and [2 litre](http://www.amazon.co.uk/Luminarc-Glass-Quadro-Fridge-Litre/dp/B00803C9L2/ref=sr_1_15?ie=UTF8&qid=1442442129&sr=8-15&keywords=glass+fridge+jug) (**£8.01** each) **all fit on either shelf**.  
_Note: On all of these, the **handle faces sideways**, parallel to the fridge door, and **the lids are not airtight** so they probably won't do as good a job of sealing in freshness as a mason jar_.

The only airtight ones I've found are plastic and some are at [ridiculous prices](http://www.amazon.co.uk/Takeya-Airtight-Pitcher-2-Quart-Blue/dp/B00DCYNP2G/ref=pd_sim_sbs_86_2?ie=UTF8&refRID=1GXS8RA6JPRF5534X6VV&dpID=31Ex2QlCRHL&dpSrc=sims&preST=_AC_UL160_SR160%2C160_).

Should we get some?
